app/controllers/admin/pages_controller.rb

```diff
@@ -0,0 +1,64 @@
+# encoding: UTF-8
+class Admin::PagesController < ApplicationController
+  load_permissions_and_authorize_resource find_by: :url
+  before_action :authorize, except: [:edit, :update]
+
+  def index
+    @page_grid = initialize_grid(Page, include: 'council')
+  end
+
+  def new
+    @page = Page.new
+  end
+
+  def edit
+    @page = Page.find_by!(url: params[:id])
+    authorize!(:manage, @page)
+  end
+
+  def create
+    @page = Page.new(page_params)
+    if @page.save
+      redirect_to edit_admin_page_path(@page), notice: alert_create(Page)
+    else
+      render :new, status: 422
+    end
+  end
+
+  def update
+    @page = Page.find_by!(url: params[:id])
+    authorize!(:manage, @page)
```

Comment:
Prefer not to use "manage" as a checking permission. Manage means "every current and future permission on this object". It's better to check against a specific action, in this case perhaps "modify".

You can also just invent a string that fits the case. If they have "manage" they also automatically have "permission-i-just-invented".